src/Workspaces/Core/Portable/Shared/Utilities/StringBreaker.cs

```diff
@@ -275,13 +275,14 @@ private static TextSpan ScanWordRun(string identifier, int length, int wordStart
                 {
                     // hit the 'o' in XMLDo.  Return "XML"
                     Debug.Assert(char.IsUpper(identifier[current - 1]));
-                    return TextSpan.FromBounds(wordStart, current - 1);
```

Comment:
❕ This form has reduced cognitive burden in cases where the computed values are start/end instead of start/length (applies to all three cases from this commit).